### DISCUSSION TITLE
error handling url - folder permissions?

### DISCUSSION THREAD

**Discussion by elinor-reid** [SchoofsKelvin/vscode-sshfs#365] [Q&A] [answered]
Hi all :)
I have a connection which works with sudo su when I run remote terminal, trying to add a folder from the remote host to the workspace fails with the above error.

Does the folder adding to workspace use the ssh user without using the sudo su?

Thanks!

**Comment by SchoofsKelvin** [comment, marked as answer]
Yes, it uses the user that is logged in over SSH, i.e. using the username you specified. You can use the `SFTP Sudo` and/or `SFTP Command` options (otherwise see #28 and/or #104) which should allow you to start the SFTP part as root.
